apps/www/pages/changelog.tsx

```diff
@@ -165,43 +165,51 @@ export const getServerSideProps: GetServerSideProps = async ({ res, query }) =>
   // Process discussions
   const formattedDiscussions = await Promise.all(
     discussions.map(async (item: any): Promise<any> => {
-      const discussionsMdxSource: MDXRemoteSerializeResult = await mdxSerialize(item.body)
-      // Find a date rewrite for the current item's title
-      const dateRewrite = deletedDiscussions.find((rewrite) => {
-        return item.title && rewrite.title && item.title.includes(rewrite.title)
-      })
-
-      // Use the createdAt date from dateRewrite if found, otherwise use item.createdAt
-      const created_at = dateRewrite ? dateRewrite.createdAt : item.createdAt
-
-      return {
-        ...item,
-        source: discussionsMdxSource,
-        type: 'discussion',
-        created_at,
-        url: item.url,
+      try {
+        const discussionsMdxSource: MDXRemoteSerializeResult = await mdxSerialize(item.body)
+        // Find a date rewrite for the current item's title
+        const dateRewrite = deletedDiscussions.find((rewrite) => {
+          return item.title && rewrite.title && item.title.includes(rewrite.title)
+        })
+
+        // Use the createdAt date from dateRewrite if found, otherwise use item.createdAt
+        const created_at = dateRewrite ? dateRewrite.createdAt : item.createdAt
+
+        return {
+          ...item,
+          source: discussionsMdxSource,
+          type: 'discussion',
+          created_at,
+          url: item.url,
+        }
+      } catch (err) {
+        console.error(`Problem processing discussion MDX: ${err}`)
```

Comment:
We should probably get this into Sentry and catching+console logging would make this a bit invisible

You could explicitly do `Sentry.captureException` - not sure about best practice here though